### ISSUE TITLE
NosTale problem with mouse.

### ISSUE THREAD

**Issue by ghost** [Frogging-Family/wine-tkg-git#508]
Hey.
There's an issue in NosTale, namely the mouse controls doesn't work quite right (buggy clicks, can't rotate the view) making it unplayable.
In some 5.x versions of Wine it was okay.

wine-tkg:
5.18.r3 - fine +
5.5.r2 - Fatal Error (launcher)
5.5.r0 - Fatal Error
5.10.r0 - Fatal Error
5.18.r3 - fine +
5.19.r1 - fine +
5.21.r0 - main issue <-
On Lutris Wine Github they wrote:
"Re-enabled dinput-remap-joystick, dinput-reconnect-joystick, dinput-axis-recalc patchsets and re-enabled gamepad functionality." and it might be because of this.
5.22.r2 - main issue
6.17 - main issue

**Comment by Tk-Glitch** [comment]
That's an upstream issue afaik. I can't find the issue we had about it anymore for some reason. Those re-enabled patchsets on lutris github are staging patchsets. It might be worth trying with a mainline build to see if it's any better.

**Comment by kajlamo** [comment]
I tried with 6.16 and 6.17 (non-tkg/staging) unfortunately same issue :/

**Comment by kajlamo** [comment]
I searched more and this problem has come up many times, was fixed, but then reappeared and so on.
2009 https://bugs.winehq.org/show_bug.cgi?id=18499 https://www.codeweavers.com/compatibility/crossover/forum/nostale (to 2017)
2010 https://bugs.winehq.org/show_bug.cgi?id=21739
2011 https://forum.winehq.org/viewtopic.php?t=11087 https://appdb.winehq.org/objectManager.php?sClass=version&iId=9349&iTestingId=67829
2017 https://forum.winehq.org/viewtopic.php?f=2&t=29682 https://bugs.winehq.org/show_bug.cgi?id=42999
2016 https://www.playonmac.com/en/topic-14836-Mouse_problem_with_nostale.html
2020 https://github.com/doitsujin/dxvk/issues/1787

**Comment by jona-1993** [comment]
I had the same issue randomly in the past with the old launcher. To fix that, I had to leave the launcher in the background.
Now, this solution doesn't work. I don't know if it's the same bug (If it can give some help).

**Comment by HoshiYamazaki** [comment]
Hey, so... Basically I pasted new logs into the wine bugzilla reports releated to this issue.

I think it's releated to the way wine communicates with Xorg, also the bug exists on macOS since always (it was not working prior to 5.20/5.21 wine release). I know that I was able to make it working by just switching window beetwen displays on Sway in 2017/2018? So maybe it's caused by some odd display logic.

Hope you guys will try to fix that.
